qcodes/plots/pyqtgraph.py

```diff
@@ -14,7 +14,7 @@
 from pyqtgraph.multiprocess.remoteproxy import ClosedError, ObjectProxy
 
 import qcodes
-import qcodes.utils.helpers
```

Comment:
a quick look at this code makes me thing that qcodes.utils are not used in this module, and then also the `self.qc_helpers`/`cls.qc_helpers` is not used at all, so perhaps we can just remove this import? or maybe i am wrong?

Reply:
Lets keep it here but not make it part of the public api